# GitHub Repository for
## Comparing gut resistome composition among patients with acute Campylobacter infections and healthy family members

### Zoe A Hansen, Wonhee Cha, Brian Nohomovich, Duane W. Newton, Paul Lephart, Hossein Salimnia, Walid Khalife, Ashley Shade, James T. Rudrik, Shannon D. Manning

### Data
Raw data for this study are available on NCBI under the BioProject PRJNA660443

### Citation 
Hansen, Z.A., Cha, W., Nohomovich, B. et al. Comparing gut resistome composition among patients with acute Campylobacter infections and healthy family members. Sci Rep 11, 22368 (2021). https://doi.org/10.1038/s41598-021-01927-7

### Abstract
Campylobacter commonly causes foodborne infections and antibiotic resistance is an imminent concern. It is not clear, however, if the human gut ‘resistome’ is affected by Campylobacter during infection. Application of shotgun metagenomics on stools from 26 cases with Campylobacter infections and 44 healthy family members (controls) identified 406 unique antibiotic resistance genes (ARGs) representing 153 genes/operons, 40 mechanisms, and 18 classes. Cases had greater ARG richness (p < 0.0001) and Shannon diversity (p < 0.0001) than controls with distinct compositions (p = 0.000999; PERMANOVA). Cases were defined by multidrug resistance genes and were dominated by Proteobacteria (40.8%), specifically those representing Escherichia (20.9%). Tetracycline resistance genes were most abundant in controls, which were dominated by Bacteroidetes (45.3%) and Firmicutes (44.4%). Hierarchical clustering of cases identified three clusters with distinct resistomes. Case clusters 1 and 3 differed from controls containing more urban and hospitalized patients. Relative to family members of the same household, ARG composition among matched cases was mostly distinct, though some familial controls had similar profiles that could be explained by a shorter time since exposure to the case. Together, these data indicate that Campylobacter infection is associated with an altered resistome composition and increased ARG diversity, raising concerns about the role of infection in the spread of resistance determinants.

### Funding
This work was supported by the National Institutes of Health [grant number U19AI090872 to S.D.M. and J.T.R.]. Salary support was provided by the United States Department of Agriculture [grant numbers 2019-67017-29112, MICL02475 to S.D.M.] and the Michigan State University (MSU) Foundation (to S.D.M.). Student support for Z.A.H. was provided by an Educational Opportunity Fellowship from the MSU Graduate School, the Ronald and Sharon Rogowski Fellowship, and the Thomas S. Whittam Graduate Award from the Department of Microbiology and Molecular Genetics at MSU as well as the James M. Tiedje travel award from the Department of Plant, Soil, and Microbial Sciences and at MSU.

### Acknowledgements
We thank Ben Hutton and Jason Wholehan at the MDHHS for help with specimen processing and culture as well as Rebekah E. Mosci, Katherine Jernigan, and Samantha Carbonell, who also helped with community DNA isolation and sequencing. We thank Drs. James M Tiedje and Lixin Zhang for their helpful discussions about these data.
